build.go

```diff
@@ -380,3 +398,27 @@ func contains(candidates []string, s string) bool {
 
 	return false
 }
+
+func ValidateSBOMFormats(layersPath string, acceptedSBOMFormats []string) error {
```

Comment:
```suggestion
func validateSBOMFormats(layersPath string, acceptedSBOMFormats []string) error {
```

Reply:
Can we make this private?

Reply:
Technically yes. It makes testing much harder though because it's being used in build, so you have to set up and run the entire build to test it. Given what it's doing, I wanted to make sure the test coverage was solid so I opted to leave it public.

Reply:
We could make it internal then? I would prefer to reduce our support surface area in terms of the public api to things that are strictly necessary - it makes it easier to modify/update code without breaking dependents.

Reply:
@dmikusa-pivotal couldn't find a good way either. Ended up just making it private and testing it the long way in #98. If that looks good, let's merge it.